homeassistant/components/splunk/__init__.py

```diff
@@ -75,13 +58,11 @@ def setup(hass, config):
     name = conf.get(CONF_NAME)
     entity_filter = conf[CONF_FILTER]
 
-    if use_ssl:
-        uri_scheme = "https://"
-    else:
-        uri_scheme = "http://"
+    hec = http_event_collector(token, host, "json", name, port, use_ssl)
```

Comment:
I would rename this variable to make it more descriptive:
```suggestion
    event_collector = http_event_collector(token, host, "json", name, port, use_ssl)
```